ui/src/shared/components/Tabs.tsx

```diff
@@ -4,11 +4,11 @@ import classnames from 'classnames'
 
 interface TabProps {
   children: JSX.Element[] | JSX.Element
-  onClick: () => void
-  isDisabled: boolean
-  isActive: boolean
-  isKapacitorTab: boolean
-  isConfigured: boolean
+  onClick?: () => void
```

Comment:
seems like all of these being conditional is a bit funky

Reply:
see above

Reply:
responded above as well